src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -71,11 +74,45 @@ namespace {{name}} {
 namespace Commands {
 {{/first}}
 namespace {{asUpperCamelCase name}} {
-struct Type {};
+enum class Fields {
+    {{#zcl_command_arguments}}
+    k{{asUpperCamelCase label}} = {{index}},
```

Comment:
Why the "k" prefix?  It mostly feels like noise at the places where it's used....